### DISCUSSION TITLE
Auto-focus the search field in Fields and Entry Types index pages

### DISCUSSION THREAD

**Discussion by thupsi** [craftcms/cms#14238] [Ideas] [answered]
Given the removal of field groups, the globalization of entry types, and the move from matrix blocks to entry types, the new search bar in Fields and Entry Types index pages was a welcome addition. 👍

Could/should maybe the search field be focused on page load? I find my self using it almost every time I visit those indexes now.

**Comment by brandonkelly** [comment, marked as answer]
Just tagged 5.0.0-alpha.11 with autofocus for those search inputs 🎉  (#14255)
